base/cvd/cuttlefish/host/commands/cvd/selector/instance_record.h

```diff
@@ -126,6 +81,13 @@ class LocalInstance {
    * instance
    */
   std::string per_instance_name_;
+
+  // Information about the group
```

Comment:
Can this be a `InstanceGroup` rather than duplicating all the same set of fields?

Reply:
Done